src/main/java/net/fabricmc/loom/providers/MinecraftMappedProvider.java

```diff
@@ -40,6 +42,12 @@
 import net.fabricmc.loom.util.DependencyProvider;
 
 public class MinecraftMappedProvider extends DependencyProvider {
+	private static final Map<String, String> jsrToJetbrains = new ImmutableMap.Builder<String, String>()
```

Comment:
```suggestion
	private static final Map<String, String> JSR_TO_JETBRAINS = new ImmutableMap.Builder<String, String>()
```

Reply:
guess so, this map is effectively immutable